README.md

```diff
@@ -26,9 +26,9 @@ To start a node independently, you need basic skills of working with the operati
 PocketnetCore is distributed in two ways: binary installer and build from source code.
 
 Minimum system requirements:
-- 4 core CPU
-- 20GB RAM
-- 25GB free disk space
+- 2 core CPU
+- 500MB RAM
```

Comment:
500MB seems very low.  Looking at the memory footprint of the process I would estimate we would want at least 4GB minimum to allow for scaling of the service over the next year.

Reply:
Ok